src/__tests__/redirect.test.ts

```diff
@@ -110,6 +110,80 @@ describe('js redirects', () => {
     expect(json.metrics.timings.close).toBeGreaterThanOrEqual(0);
   });
 
+  it('should catch path', async () => {
+    const { res, body } = await postRender({
+      url: 'http://localhost:3000/test-website/js-redirect-path.html',
+      ua: 'Algolia Crawler',
+      waitTime: {
+        min: 2000,
+      },
+    });
+
+    const json: PostRenderSuccess = JSON.parse(body);
+    expect(res.statusCode).toBe(200);
+
+    expect(json.statusCode).toBe(200);
+    expect(json.body).toBeNull();
+    expect(json.resolvedUrl).toBe(
+      'http://localhost:3000/test-website/basic.html'
+    );
+    expect(json.error).toBe('redirection');
```

Comment:
Weird to have it in `error`

Reply:
It's in error because we don't generate a body in that case.
maybe this field should have been named `status` instead.